packages/dd-trace/src/proxy.js

```diff
@@ -47,7 +51,26 @@ class Tracer extends NoopProxy {
         // dirty require for now so zero appsec code is executed unless explicitly enabled
         if (config.appsec.enabled) {
           require('./appsec').enable(config)
+        } else if (config.appsec.enabled === undefined) { // only activate remote config when conf is not set locally
+          rc.updateCapabilities(RemoteConfigCapabilities.ASM_ACTIVATION, true)
+
+          rc.on('ASM_FEATURES', (action, conf) => {
```

Comment:
Can this be moved elsewhere? It doesn't really belong in the proxy.

Reply:
You're right, what do you think about moving this to appsec's index.js ? that would mean switching from the "dirty require" that we have right now to prevent any appsec code to be run unless specifically enabled, to a normal top of file require.

Reply:
Seems to me more like it should live in the config code itself. 🤔

Reply:
I've moved it not in the config.js file as that would be too akward imo, but it it's own index.js under the remote_config folder. We can always change it later but for now since RC is only used by appsec i think it makes sense.